packages/prbnmcn-dagger/prbnmcn-dagger.0.0.1/opam

```diff
@@ -9,9 +9,13 @@ license: "MIT"
 homepage: "http://github.com/igarnier/prbnmcn-dagger"
 bug-reports: "http://github.com/igarnier/prbnmcn-dagger"
 depends: [
-  "dune" {>= "2.8" & >= "2.8"}
+  "dune" {>= "2.9"}
   "pringo" {>= "1.3"}
   "prbnmcn-cgrph" {= "0.0.2"}
+  "qcheck" {>= "0.17" & with-test}
+  "prbnmcn-dagger-gsl" {= "0.0.1" & with-test}
```

Comment:
To make this kind of circular dependencies work, you should just add `post`:
```
"prbnmcn-dagger-gsl" {= "0.0.1" & with-test & post}
```

(same for the two lines below)

Reply:
Oh, good to know. Thanks for the feedback!